Library/Formula/rtl_433.rb

```diff
@@ -0,0 +1,19 @@
+class Rtl433 < Formula
+  desc "Tool to talk to 433Mhz devices using RTL devices"
+  homepage "https://github.com/merbanan/rtl_433"
+  url "https://github.com/merbanan/rtl_433.git", :revision => "922a76a81820c68e2e5c55a428e5d22135a8be9a"
```

Comment:
We'll need a stable, tagged upstream release here. You'll need to ask them for one, I suspect.